src/kubernetes-ingress-controller/guides/using-kongplugin-resource.md

```diff
@@ -421,44 +508,64 @@ metadata:
 username: harry
 credentials:
 - harry-apikey" | kubectl apply -f -
+```
+
+```
 kongconsumer.configuration.konghq.com/harry configured
 ```
 
 Note the annotation being added to the `KongConsumer` resource.
 
 Now, if the request is made as the `harry` consumer, the client
-will be rate-limited differently:
+is rate-limited differently:
+
+```sh
+curl -I $PROXY_IP/foo -H 'apikey: my-sooper-secret-key'
+```
 
-```bash
-$ curl -I $PROXY_IP/foo -H 'apikey: my-sooper-secret-key'
+```
 HTTP/1.1 200 OK
 Content-Type: text/html; charset=utf-8
 Content-Length: 9593
 Connection: keep-alive
+X-RateLimit-Remaining-Minute: 9
+X-RateLimit-Limit-Minute: 10
+RateLimit-Remaining: 9
+RateLimit-Reset: 42
+RateLimit-Limit: 10
 Server: gunicorn/19.9.0
 Access-Control-Allow-Origin: *
 Access-Control-Allow-Credentials: true
-X-RateLimit-Limit-minute: 10
-X-RateLimit-Remaining-minute: 9
 demo:  injected-by-kong
-X-Kong-Upstream-Latency: 3
+X-Kong-Upstream-Latency: 2
 X-Kong-Proxy-Latency: 1
-Via: kong/1.2.1
+Via: kong/2.8.1
+```
 
-# a regular unauthenticated request
-$ curl -I $PROXY_IP/bar
+And a regular unauthenticated request:
+
+```sh
+curl -I $PROXY_IP/bar
+```
+
+```sh
 HTTP/1.1 200 OK
 Content-Type: text/plain; charset=UTF-8
 Connection: keep-alive
+X-RateLimit-Remaining-Minute: 4
+X-RateLimit-Limit-Minute: 5
+RateLimit-Remaining: 4
+RateLimit-Reset: 11
+RateLimit-Limit: 5
 Server: echoserver
-X-RateLimit-Limit-minute: 5
-X-RateLimit-Remaining-minute: 4
 demo:  injected-by-kong
-X-Kong-Upstream-Latency: 2
+X-Kong-Upstream-Latency: 1
 X-Kong-Proxy-Latency: 1
-Via: kong/1.2.1
+Via: kong/2.8.1
 ```
 
-This guide demonstrates how you can use the {{site.kic_product_name}} to
-impose restrictions and transformations
-on various levels using Kubernetes style APIs.
+There is a lot more you can do with Kong Plugins, check the [Plugin Hub](/hub) for
+more information.
+
+Next, you might want to learn more about Ingress with the 
+[KongIngress resource guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-kongingress-resource/).
```

Comment:
```suggestion
## Next steps

There's a lot more you can do with Kong plugins. Check the [Plugin Hub](/hub) to see all of the available plugins and how to use them.

Next, you might want to learn more about Ingress with the 
[KongIngress resource guide](/kubernetes-ingress-controller/{{page.kong_version}}/guides/using-kongingress-resource/).
```

Minor phrasing edit, mainly wanted to add the header. Also, contractions are totally fine to use (preferred, even).